### ISSUE TITLE
PM's should not be reported to HipChat

### ISSUE THREAD

**Issue by koinu** [binaryage/discourse-hipchat-plugin#1]
This plugin is sending notifications to HipChat when private messages are sent. (Observed with Discourse v1.4.0.beta2 +2 and version 1.0 of this plugin.)

In the case of new user sign-ups and the "Welcome to {{forum}}!" PM, it's almost useless because only admins can actually click the link to see who the new user is (by way of seeing who the PM was sent to). In all other cases (PMs between users), it's actively creepy to see a feed of even just the titles.

**Comment by darwin** [comment]
I agree and confirm the behaviour. But this is not a big issue for my use-case. Don't expect me to fix it soon.

Patches welcome.
